### ISSUE TITLE
convergence problem

### ISSUE THREAD

**Issue by songwendong** [weinman/cnn_lstm_ctc_ocr#4]

**Comment by songwendong** [comment]
![image](https://user-images.githubusercontent.com/16954370/29446536-3f8439d0-841f-11e7-916d-4baf75b098cb.png)

**Comment by Govan111** [comment]
Do you solve the problem?I'm coping with the problem that is same with yours.

**Comment by weinman** [comment]
Are you using the mjsynth data framework and default hyperparameters (e.g., learning rate) provided or your own data, etc.?
